src/workers/sourceSquadCreated.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  getContactIdByEmail,
+  addUserToContacts,
+  LIST_DRIP_CAMPAIGN,
+} from '../common';
+import { Source, SourceMember, User, UserActionType } from '../entity';
+import { SourceMemberRoles } from '../roles';
+import { insertOrIgnoreAction } from '../schema/actions';
+import { ChangeObject } from '../types';
+import { messageToJson, Worker } from './worker';
+
+interface Data {
+  source: ChangeObject<Source>;
+}
+
+const worker: Worker = {
+  subscription: 'api.source-squad-created',
+  handler: async (message, con, logger) => {
+    const data: Data = messageToJson(message);
+
+    try {
+      const { source } = data;
+      const [owner] = await con
+        .getRepository(SourceMember)
+        .createQueryBuilder('sm')
+        .select('sm."userId" as "userId"')
+        .where('sm."sourceId" = :sourceId', { sourceId: source.id })
+        .andWhere('sm."role" = :role', { role: SourceMemberRoles.Admin })
+        .orderBy('sm."createdAt"', 'ASC')
+        .limit(1)
+        .execute();
+
+      if (!owner) {
+        return;
+      }
+
+      const user = await con
+        .getRepository(User)
+        .findOneBy({ id: owner.userId });
```

Comment:
Just wondering is this not already retriever on the owner you got above here?

Reply:
Can also do a `const user = await owner.user`.

Reply:
good point, added 2ba74f8a27743bb5e654a09fc0bf7350a5fce6ae